ChangeLog.md

```diff
@@ -1,3 +1,22 @@
+## 2017.09.27 - Azure ARM version 0.13.0
+
+* Regened all azure_mgmt_* sdks using [AutoRest.2.0.4144](https://www.npmjs.com/package/autorest) for release of ARM version 0.13.0
+* Updated [azure_mgmt_cognitive_services](https://rubygems.org/gems/azure_mgmt_cognitive_services) operations Update, Delete, GetProperties, ListKeys, RegenerateKey & ListSkus from CognitiveServiceAccounts group to Accounts group. Refer [PR #989](https://github.com/Azure/azure-sdk-for-ruby/pull/989) and [PR #1707](https://github.com/Azure/azure-rest-api-specs/pull/1707) for further details.
+* Updated [azure_mgmt_compute](https://rubygems.org/gems/azure_mgmt_compute) with Vmss rolling upgrade, Vmss patch and Vm instance view APIs. Refer [PR #999](https://github.com/Azure/azure-sdk-for-ruby/pull/999) and latest [specification](https://github.com/Azure/azure-rest-api-specs/blob/current/specification/compute/resource-manager/readme.md) for further details.
```

Comment:
the link to the "specification" will always point to latest, if you wanted to point to the state today, you may need to get a "perma link" or a hash for the commit that you're referencing. 
I would actually recommend to add a link with the hash to the last commit you got, for the whole repo, so we know which snapshot we took. Also include the version of autorest and autorest.ruby being used.